pinot-orc/src/main/java/org/apache/pinot/orc/data/readers/ORCRecordReader.java

```diff
@@ -164,6 +165,8 @@ private Object getBaseObject(WritableComparable w) {
       obj = ((DoubleWritable) w).get();
     } else if (BytesWritable.class.isAssignableFrom(w.getClass())) {
       obj = ((BytesWritable) w).getBytes();
+    } else if (ByteWritable.class.isAssignableFrom(w.getClass())) {
```

Comment:
Why `toString()`?

Reply:
the value is 8-bits tinyint, both `get()` and `toString()` will give same human readable numbers in a string format.

Reply:
Can you put this after the boolean type for readability?

Reply:
will do